python/flexflow_dataloader.cc

```diff
@@ -270,49 +269,6 @@ void SingleDataLoader::next_batch_xd_launcher(FFModel &ff, int task_id) {
     runtime->execute_index_space(ctx, launcher);
   }
   next_index += ff.config.batchSize;
-#else
```

Comment:
@eddy16112 Can you create a new issue then for us to look into this with ideally a brief writeup of the context of this code and enything else you remember (and link it in a reply to this comment)? While it's fine for this to stay in for now it would be good to get it cleaned up eventually

Reply:
@eddy16112 Reminder when you get a chance